decision-making.md

```diff
@@ -1,8 +1,8 @@
 #Decision-making
 
 There are at least two popular aproaches:
-* Do-ocracy is an organizational way where every individual chooses own role and/or task and does it autonomously or in a team. But accontability sticks to actor who implement task/role but not who is officially assigned. 
-  * The main advantages of this approach: big motivation and rapid implementation. 
+* Democracy is an organizational way where every individual chooses own role and/or task and does it autonomously or in a team. But accountability sticks to actor who implement task/role but not who is officially assigned. 
```

Comment:
Do-ocracy is different from democracy so if you could undo this change I'd be delighted to merge.